core/src/avm1/property.rs

```diff
@@ -40,13 +40,10 @@ impl<'gc> Property<'gc> {
     /// discarded.
     pub fn set(&mut self, new_value: impl Into<Value<'gc>>) -> Option<Object<'gc>> {
         match self {
-            Property::Virtual { set, .. } => {
-                if let Some(function) = set {
-                    Some(function.to_owned())
-                } else {
-                    None
```

Comment:
The old one honestly feels more readable to me.

Reply:
Agree, but this function will be completely removed in #4873.

Reply:
Cool. Then let's get this merged to unblock other stuff